lib/src/main/kotlin/starknet/extensions/json.kt

```diff
@@ -0,0 +1,10 @@
+package starknet.extensions
+
+import kotlinx.serialization.json.*
+import starknet.data.types.Felt
+
+fun JsonObjectBuilder.putFeltAsHex(key: String, value: Felt?): JsonElement? =
+    put(key, JsonPrimitive(value?.hexString() ?: Felt.ZERO.hexString()))
+
+fun JsonArrayBuilder.addFeltAsHex(value: Felt?): Boolean =
+    add(JsonPrimitive(value?.hexString() ?: Felt.ZERO.hexString()))
```

Comment:
Is it more readable than value.hexString? We could turn hexString into a property.
```
put(key, value.hexString)
```
vs
```
putFeltAsHex(key, value)
```
I see you use a fallback for value, but what is the use case? Is it safe for us to fallback to 0x0? I am afraid in some cases it might be not what we want.

Reply:
I guess we can just not support nullable values in these methods and we won't need to have a fallback value.

Reply:
And I think it is just a little bit more readable since it makes result of the function call immediately obvious unlike the cast

Reply:
I don't see why value.hexString would be less readable in terms of the output. 
Do we need to use decimals for felts somehwere? If not then what if we added a `put` extension that accepts Felt and puts hex in? This way we extend offered native api.

Reply:
I mean, we can just name it `put`. That's what I initially did but then changed it to make it more obvious. It doesn't change much on the user side, since they have to import this function anyway, no matter if it's named `put` or not.

Reply:
I think we should either name it as native methods are named or not add it at all.